airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -17,4 +17,20 @@ $(document).ready(function () {
   })
   // Initialize the editor with the div id and path to Popcorn Editor.
   PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
+  PopcornEditor.listen('save', function (message) {
+    console.log(message);
+    $.post($('#editor').data('save'), {
+      data: JSON.stringify(message.data),
+      slug: $('#editor').data('slug'),
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val()
+    })
+    .done(function (response) {
+      alert('Event is saved and processing with PopcornEdit.id = '
+           + response.id);
```

Comment:
Any ideas? I think having an alert saying your data has been saved is less descriptive and really this will need a fancy modal for it to really be usable. I wanted to keep it out of this PR just so we could troubleshoot the whole process as soon as possible.